GOVERNANCE.md

```diff
@@ -0,0 +1,145 @@
+TC for discussion by assigning the ***tc-agenda*** tag to a pull
+request or issue. The TC should serve as the final arbiter where
+required.
+
+For the current list of Collaborators, see the project
+[README.md](./#current-project-team-members).
+
+A guide for Collaborators is maintained in
+[COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
+
+## TC Membership
+
+TC seats are not time-limited.  There is no fixed size of the TC.
+However, the expected target is between 6 and 12, to ensure adequate
+coverage of important areas of expertise, balanced with the ability to
+make decisions efficiently.
+
+There is no specific set of requirements or qualifications for TC
+membership beyond these rules.
+
+The TC may add additional members to the TC by unanimous consensus.
+
+A TC member may be removed from the TC by voluntary resignation, or by
+unanimous consensus of all other TC members.
+
+Changes to TC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "TC Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
+TC is not in attendance to participate, then the addition or removal
+is added to the agenda for the subsequent meeting.  This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions.  If a TC member is unable to attend a meeting
+where a planned membership decision is being made, then their consent
+is assumed.
+
+No more than 1/3 of the TC members may be affiliated with the same
+employer.  If removal or resignation of a TC member, or a change of
+employment by a TC member, creates a situation where more than 1/3 of
+the TC membership shares an employer, then the situation must be
+immediately remedied by the resignation or removal of one or more TC
+members affiliated with the over-represented employer(s).
+
+## TC Meetings
+
+The TC meets weekly on a Google Hangout On Air. The meeting is run by
+a designated moderator approved by the TC. Each meeting should be
+published to YouTube.
+
+Items are added to the TC agenda which are considered contentious or
+are modifications of governance, contribution policy, TC membership,
+or release process.
+
+The intention of the agenda is not to approve or review all patches,
+that should happen continuously on GitHub and be handled by the larger
+group of Collaborators.
+
+Any community member or contributor can ask that something be added to
+the next meeting's agenda by logging a GitHub Issue. Any Collaborator,
+TC member or the moderator can add the item to the agenda by adding
+the ***tc-agenda*** tag to the issue.
+
+Prior to each TC meeting the moderator will share the Agenda with
+members of the TC. TC members can add any items they like to the
+agenda at the beginning of each meeting. The moderator and the TC
+cannot veto or remove items.
+
+The TC may invite persons or representatives from certain projects to
```

Comment:
I'd like to have a provision for adding *voting* members from these groups as well. I don't know how we want to handle that though, do we just want to nominate them as a regular member or do we want to hand a vote to these sub-groups?